code/game/objects/items/weapons/blades.dm

```diff
@@ -42,16 +42,23 @@
 	return(BRUTELOSS)
 
 /obj/item/weapon/claymore/harvester
+	parent_type = /obj/item/weapon/twohanded
```

Comment:
This wreaks of shitcode. Ima have to ask about this. Does it work?

Reply:
yes i tested it and it works. Other way would be moving whole harvester blade code into twohanded and then fixing all typepaths